airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/streams/base_insight_streams.py

```diff
@@ -43,6 +43,7 @@ class AdsInsights(FBMarketingIncrementalStream):
         "action_target_id",
         "action_destination",
     ]
+    action_report_time = "mixed"
```

Comment:
better use different variable name from class instance argument 

`default_action_report_time` can be good candidate